src/styles.css

```diff
@@ -0,0 +1,214 @@
+* {
```

Comment:
let's keep here global styles, and extract all components related styles

Reply:
as we discussed during the meeting it will be fixed in the next MR